### PR TITLE
Add ⭐️

### PR DESCRIPTION
**Pull request by GitHubGW**
### Sub

### CODE REVIEW

File: add.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
+⭐️⭐️⭐️⭐️⭐️
```
Comment (SugarGW) [inline add.txt:1]: Too many stars..

File: add.txt [hunk 2]
```diff
@@ -1,2 +1,4 @@
...
 local2
 remote2
+⭐️⭐️⭐️⭐️⭐️
```
Comment (SugarGW) [inline add.txt:4]: Please leave only one star

**Review by SugarGW — APPROVED** [review]
Thank you!

File: add.txt (hunk 1 above)
Comment (GitHubGW) [reply add.txt:1]: okay

File: add.txt (hunk 2 above)
Comment (GitHubGW) [reply add.txt:4]: got it!

**Pull request merged**